tests/test_compression_algorithms.py

```diff
@@ -0,0 +1,48 @@
+from src.algorithmForNN.compressUtils import FIC, LZW
+
+import os
+
+import pytest
+
+
+def test_compress_decompress_LZW_text_from_file():
+    path_to_file = "tests/data/ownData.txt"
+    with open(path_to_file, "r") as file:
+        start_text = file.read()
+        end_text = LZW.decompress(LZW.compress(start_text))
+
+        assert start_text == end_text
+
+
+def test_compress_decompress_FIC_list():
+    start_list = [1, 4, 3, 250, 4056, 10, 128, 1024, 253, 13, 53]
+    end_list = FIC.decompress(FIC.compress(start_list))
+
+    assert start_list == end_list
+
+
+def test_full_compress_decompress_simple_string():
+    start_string = "Skoltech is the best"
+    end_string = LZW.decompress(
+        FIC.decompress(FIC.compress(LZW.compress(start_string)))
+    )
+
+    assert start_string == end_string
+
+
+def test_compress_decompress_text_from_file():
```

Comment:
Name this method better, as I understood it is connected to compatibility between resulted files, so show this in the name